src/main/java/frc5124/robot2020/subsystems/Camera.java

```diff
@@ -8,14 +8,20 @@
 package frc5124.robot2020.subsystems;
 
 import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc5124.robot2020.RobotMap;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
 
 public class Camera implements Subsystem {
+  private NetworkTableEntry shuffleboardButtonBooleanEntry;
+  private ShuffleboardTab display;
   
   public Camera() {
   }
 
   @Override
   public void periodic() {
+    if (RobotMap.debugEnabled) {}
```

Comment:
what is the purpose of always checking if debug enabled and then doing nothing in response?